### PR TITLE
move common api creds fetching code to layer

### PR DESCRIPTION
**Pull request by attwad**

### CODE REVIEW

File: ingestion/functions/common/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+boto3==1.13.26
```
Comment (axmb) [inline ingestion/functions/common/requirements.txt:1]: Is this one needed?

File: ingestion/functions/parsing/ch_zurich/input_event.json [hunk 2]
```diff
@@ -1,10 +1,6 @@
 {
     "s3Bucket": "epid-sources-raw",
-    "s3Key": "5f32be3fe3f3ef002e849307/2020/08/11/1552/content.csv",
+    "s3Key": "5f32be3fe3f3ef002e849307/2020/08/12/0815/content.csv",
     "sourceUrl": "https://github.com/openZH/covid_19/blob/master/fallzahlen_kanton_alter_geschlecht_csv/COVID19_Fallzahlen_Kanton_ZH_alter_geschlecht.csv",
-    "sourceId": "5f32be3fe3f3ef002e849307",
-    "dateFilter": {
```
Comment (axmb) [inline ingestion/functions/parsing/ch_zurich/input_event.json:6]: Was the dateFilter intentionally dropped?

File: ingestion/functions/common/requirements.txt (hunk 1 above)
Comment (attwad) [reply ingestion/functions/common/requirements.txt:1]: For s3 client yes

File: ingestion/functions/parsing/ch_zurich/input_event.json (hunk 2 above)
Comment (attwad) [reply ingestion/functions/parsing/ch_zurich/input_event.json:6]: Was just testing, no big deal if it's dropped or not.

**Pull request merged**